.github/workflows/publish_npm.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         workflow: manifold.yml
         workflow_conclusion: completed
         branch: master
-        check_artifact: true
+        check_artifacts: true
```

Comment:
@pca006132 any reason we should use `branch: master` here instead of `run_id` like above?

Reply:
`publish_npm` is not triggered by another workflow, so we can't do that.